pkg/k8s/chart/types.go

```diff
@@ -0,0 +1,20 @@
+package chart
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+//go:generate mockery --name=Renderer --outpkg=mocks --case=underscore
+type Renderer interface {
+	// RenderManifest of the given chart.
+	RenderManifest(*ReleaseInstance) (string, error)
+
+	// RenderManifestAsUnStructured of the given chart as unstructured objects.
+	RenderManifestAsUnStructured(*ReleaseInstance) (*ManifestResources, error)
+}
+
+// ManifestResources holds a collection of objects.
+type ManifestResources struct {
+	Items []*unstructured.Unstructured
+	Blobs [][]byte
```

Comment:
Same as https://github.com/kyma-project/nats-manager/pull/17/files#r1152007417.

Reply:
I only used the struct to pass data among methods. But yes, valid point. I will do it in later PR, as it may fail some things in the parent PR. I have added a task [here](https://github.com/kyma-project/nats-manager/issues/8) for it.